data/src/test/java/com/android254/data/repos/SessionsManagerTest.kt

```diff
@@ -21,21 +21,20 @@ import androidx.test.ext.junit.runners.AndroidJUnit4
 import com.android254.data.dao.BookmarkDao
 import com.android254.data.dao.SessionDao
 import com.android254.data.db.Database
+import com.android254.data.db.model.SessionEntity
 import com.android254.data.network.apis.SessionsApi
 // import com.android254.domain.models.DataResult
 // import com.android254.domain.models.Success
```

Comment:
We should clean this up if they're not being used

Reply:
Noted